packages/core/src/enqueue_links/enqueue_links.ts

```diff
@@ -117,11 +128,30 @@ export async function enqueueLinks(options: EnqueueLinksOptions): Promise<QueueO
             ow.object.validate(validators.pseudoUrl),
         ))),
         transformRequestFunction: ow.optional.function,
+        strategy: ow.optional.string.oneOf([
+            DomainStrategy.All,
```

Comment:
maybe it would make more sense to call it `EnqueueStrategy`?

Reply:
Agreed, will change 👍